plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/FeatureDiagramEditor.java

```diff
@@ -728,8 +728,6 @@ else if ((hiddenAction.isEnabled()) && !connectionSelected) {
 				extension.extendContextMenu(menu, this);
 			}
 		}
-		menu.add(new Separator(IWorkbenchActionConstants.MB_ADDITIONS));
-		menu.add(this.exportFeatureModelAction);
```

Comment:
This change removes the Export As entry from almost all context menus. This is not intended.